innhold/kap1/README.md

```diff
@@ -7,9 +7,11 @@ Her er det bare å starte fra kapittel 1.1 og jobbe seg gjennom kapitlene, bit f
 Om dette er stoff du kjenner til fra før kan du vurdere å gå videre til [del 2](../kap2/). I så fall kan det være lurt å se raskt igjennom kapitlene for å forsikre deg om at du ikke går glipp av noe.
 
 1. [Kjøre Python-program](1_kjøre_pythonprogram.md)
+    * Hva er Python
+    * Kjøre Python interaktivt
     * Navigere i filhierarkiet
-    * Skrive ditt første Python-program
-    * Kjøre Python-program
+    * Skrive ditt første Python-skript
```

Comment:
Jeg lurer på om vi skulle ha kalt det program selv om det teknisk sett er skript, for ikke å skape forvirring om forskjellen, og skript er kanskje et ukjent ord for en del?

Reply:
Hmm. Jeg er tvert imot redd for at det skaper forvirring hvis vi bruker Python-program, for refererer vi til Python-fortolkeren eller til et Python-skript? Jeg tenker at uten Python foran går det greit, men skal vi kvalifisere det med Python så bør vi bruke Python-skript.

Jeg har endret så kurset ikke bruker begrepet «Python-program», og forklarer at skript og program gjerne brukes som synonymer.